src/core_modules/capture-core/components/WorkingLists/EventWorkingLists/eventWorkingLists.actions.js

```diff
@@ -15,8 +15,8 @@ export const deleteEventSuccess =
 export const deleteEventError =
     () => actionCreator(actionTypes.EVENT_DELETE_ERROR)();
 
-export const openViewEventPage =
-    (eventId: string) => actionCreator(actionTypes.VIEW_EVENT_PAGE_OPEN)({ eventId });
+export const openViewEventPage = (eventId: string, programId: string, orgUnitId?: string) =>
```

Comment:
hey, should the programId also optional because of the `program?.id`

Reply:
Good catch! I updated the code with your suggestion. Thanks!